### DISCUSSION TITLE
LGTM Custom Query Polling Behavior

### DISCUSSION THREAD

**Discussion by ayman-m** [github/securitylab#115] [General] [answered]
I have created couple of queries and placed them under {my-project-root}/lgtm/python-queries folder, but LGTM doesn't seem to process them (no custom query logs), I have found the following statement in LGTM's documentation:
**_"The daily LGTM poll jobs will detect the queries and automatically run them."_**
My understanding of the above statement is that LGTM will only process new queries once a day, if my understanding is correct. Is that something that can be changed ?

**Comment by jbj** [comment, marked as answer]
Can you provide a link to the project repo? That should help with debugging.

For the record, the relevant documentation is https://lgtm.com/help/lgtm/about-queries#how-do-i-add-custom-queries-to-LGTM.
